src/components/AccessReviewHeader.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { div, a, hh } from "react-hyperscript-helpers";
+import { Theme } from '../libs/theme';
+import { Storage } from "../libs/storage";
+import { Navigation } from "../libs/utils";
+
+const TITLE = {
+  fontWeight: Theme.font.weight.semibold,
+  fontSize: Theme.font.size.title,
+  lineHeight: Theme.font.leading.title,
+  marginBottom: '5px',
+}
+
+const SMALL = {
+  fontSize: Theme.font.size.small,
+  lineHeight: Theme.font.leading.dense,
+}
+
+const BUTTON = {
+  ...SMALL,
+  fontWeight: Theme.font.weight.semibold,
+}
+
+export const AccessReviewHeader = hh(class AccessReviewHeader extends React.PureComponent {
+
+  render() {
+    const currentUser = Storage.getCurrentUser();
+    const { history } = this.props;
+    return div(
+      {
+        style: {
+          justifyContent: "space-between",
+          alignItems: "center",
+          display: "flex",
+          fontFamily: 'Montserrat',
+          color: Theme.palette.primary,
+        }
+      },
+      [
+        div(
+          {
+            style: {
+              alignItems: "center",
+              display: "flex"
+            }
+          },
+          [
+            div({
+              id: "lock-icon",
+              style: {
+                width: "50px",
+                height: "50px",
+                backgroundColor: "#eeeeee",
+                marginRight: "16px"
+              }
+            }),
+            div({ id: "header-text" }, [
+              div({ style: TITLE }, "Data Access Review"),
+              div({ style: SMALL },
+                "Review the Application Summary and Data Use Limitations to determine if the researcher should be granted access to the data."
+              ),
+              div({ style: SMALL }, "DAC Chairs can optionally vote as a member.")
+            ])
+          ]
+        ),
+        div(
+          [
+            a(
+              {
+                id: "btn_back",
+                onClick: () => Navigation.back(currentUser, history),
+                className: "button-outlined secondary",
+                style: BUTTON,
+              },
+              "DAC Console"
```

Comment:
Can we add another button here to link back to the original version of the page? That would allow for us to vote on elections on dev while this feature is in development.